package.json

```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "acorn-globals": "4.1.0",
     "ajv": "6.1.1",
-    "ajv-keywords": "3.1.0",
+    "ajv-keywords": "2.1.1",
```

Comment:
revert this as well

Reply:
They've reverted, no longer exist in the PR